src/LegacyLinkBlockRepository.php

```diff
@@ -134,21 +136,22 @@ public function dropTables()
     public function installFixtures()
     {
         $success = true;
-        $id_hook = (int)Hook::getIdByName('displayFooter');
+        $id_hook = (int) Hook::getIdByName('displayFooter');
         $queries = [
-            'INSERT INTO `'.$this->db_prefix.'link_block` (`id_link_block`, `id_hook`, `position`, `content`) VALUES
-                (1, '.$id_hook.', 0, \'{"cms":[false],"product":["prices-drop","new-products","best-sales"],"static":[false]}\'),
-                (2, '.$id_hook.', 1, \'{"cms":["1","2","3","4","5"],"product":[false],"static":["contact","sitemap","stores"]}\');'
+            'INSERT INTO `' . $this->db_prefix . 'link_block` (`id_link_block`, `id_hook`, `position`, `content`) VALUES
+                (1, ' . $id_hook . ', 0, \'{"cms":[false],"product":["prices-drop","new-products","best-sales"],"static":[false]}\'),
+                (2, ' . $id_hook . ', 1, \'{"cms":["1","2","3","4","5"],"product":[false],"static":["contact","sitemap","stores"]}\');',
```

Comment:
The trailing `,` is valid ? Not a mistake ?

Reply:
I think php-cs-fixer added it but I don't see why it would be a problem?

Reply:
Nevermind 😅 I was mislead by the concatenation of `);',`

Reply:
yeah I thought so ^^